documentation/modules/oauth/con-mapping-keycloak-authz-services-to-kafka-model.adoc

```diff
@@ -38,10 +37,13 @@ Authorization scopes must contain all the possible Kafka permissions regardless
 * `Delete`
 * `Describe`
 * `Alter`
-* `DescribeConfig`
-* `AlterConfig`
+* `DescribeConfigs`
+* `AlterConfigs`
 * `ClusterAction`
 * `IdempotentWrite`
+* `CreateTokens`
+* `DescribeTokens`
+* `All`
```

Comment:
These are not supported. We don't support authentication tokens for authentication and we don't support `All` as a valid operation.
```suggestion
```

Reply:
For more on delegation tokens see: https://kafka.apache.org/documentation/#security_delegation_token

Reply:
Thanks @mstruk : we say this before introducing the list: "Authorization scopes must contain all the possible Kafka permissions regardless of the resource type:"

So, not quite , then?  We should remove the above and say something like: " "Authorization scopes _should_ contain the following Kafka permissions regardless of the resource type:"?

Reply:
Should we include a note on delegation tokens or is it implicit?

Reply:
The `CreateTokens` and `DescribeTokens` are redundant but by themself not harmful. Adding `All` can lead one to believe that it is a legit operation name to use in order to match any other, which is not supported.
So yes, it would be good to mention at leas the `All`, that using it is not supported.

Reply:
Thanks @mstruk. I've added a note. If you're happy with the changes, please approve.